content/blog/getting-started-with-hpe-greenlake-api-for-backup-and-recovery.md

```diff
@@ -0,0 +1,516 @@
+    {
+      "type": "SNAPSHOT",
+      "schedules": [
+        {
+          "scheduleId": 1,
+          "name": "Array_Snapshot_1",
+          "namePattern": {
+            "format": "Array_Snapshot_{DateFormat}"
+          },
+          "expireAfter": {
+            "unit": "DAYS",
+            "value": 1
+          },
+          "schedule": {
+            "recurrence": "HOURLY",
+            "repeatInterval": {
+              "every": 4
+            },
+            "activeTime": {
+              "activeFromTime": "00:00",
+              "activeUntilTime": "23:59"
+            }
+          }
+        }
+      ]
+    },
+    {
+      "type": "BACKUP",
+      "protectionStoreId": "<onprem-protection-store-id>",
+      "schedules": [
+        {
+          "scheduleId": 2,
+          "name": "On-Premises_Protection_Store_2",
+          "sourceProtectionScheduleId": 1,
+          "namePattern": {
+            "format": "On-Premises_Protection_Store_{DateFormat}"
+          },
+          "expireAfter": {
+            "unit": "DAYS",
+            "value": 3
+          },
+          "schedule": {
+            "recurrence": "DAILY",
+            "repeatInterval": {
+              "every": 1
+            },
+            "startTime": "00:00"
+          }
+        }
+      ]
+    },
+    {
+      "type": "CLOUD_BACKUP",
+      "protectionStoreId": "<cloud-protection-store-id>",
+      "schedules": [
+        {
+          "scheduleId": 3,
+          "name": "HPE_Cloud_Protection_Store_3",
+          "sourceProtectionScheduleId": 2,
+          "namePattern": {
+            "format": "HPE_Cloud_Protection_Store_{DateFormat}"
+          },
+          "expireAfter": {
+            "unit": "WEEKS",
+            "value": 1
+          },
+          "schedule": {
+            "recurrence": "DAILY",
+            "repeatInterval": {
+              "every": 2
+            },
+            "startTime": "00:00"
+          }
+        }
+      ]
+    }
+  ],
+  "applicationType": "VMWARE"
+}
+```
+
+The above figure shows JSON structure for request body of **POST /backup-recovery/v1beta1/protection-policies** for the creation of protection-policy as intended. 
+
+6. Finally, I created of the protection policies using the GreenLake API for Backup and Recovery **POST /backup-recovery/v1beta1/protection-policies**, and I used the above JSON structure in the request body. This is a special POST API execution where the response is returned immediately. The response of this API contained the body of JSON structure that will be useful to identify the protection-jobs such as the **“<protection-policies-id>”**
+
+![API to create protection policy](/img/api-to-create-protection-policy.png)
+
+7. The below figure shows the complete response JSON body from the above API that shows the construction of the protection policy with different protection tiers and the schedules associated with the protection tier. The important values were the ids for different protection tiers:
```

Comment:
Suggestion to not make the item 7 a numbered item. Just remove the "7.". 

Please check the preview: https://deploy-preview-2458--hpe-dev-portal.netlify.app/blog/getting-started-with-hpe-greenlake-api-for-backup-and-recovery/

You will see that the values for ID, Snapshot, On-premises and Cloud are listed as numbered item. They should just be listed as bullet items in the list.